deepin-font-manager/views/dfontmgrmainwindow.cpp

```diff
@@ -1160,6 +1160,14 @@ void DFontMgrMainWindow::installFontFromSys(const QStringList &files)
             return;
         } else if (m_isPopInstallErrorDialog) {
             emit m_signalManager->installDuringPopErrorDialog(reduceSameFiles);
+        } else if (m_fIsInstalling || m_fontLoadingSpinner->isVisible()) {
+            qDebug() << "Is installing ,quit";
+            foreach(auto it, reduceSameFiles) {
```

Comment:
```suggestion
            foreach (auto it, reduceSameFiles) {
```

如果 log 要留着的话，考虑给 log 补充点信息进去？不留着的话删掉就可以。

另外考虑把 `foreach` 改成 `for (type var: list)` 的形式？后者是 c++ 允许的，前者是 Qt 自己的糖。